.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+The issue tracker isn't the place for general support. For questions and
+technical assistance, come ask the
+[Bokeh mailing list](https://groups.google.com/a/continuum.io/forum/#!forum/bokeh).
+
+For defects or deficiencies, please provide:
+* Description of expected behavior and the observed behavior
+* self-contained example code that generates the issue (if possible)
+* stack traceback and/or browser console error output
```

Comment:
we probably need a link to let know the user how to look into the js console... some users does not know about that...